pkg/internal/controller/controller.go

```diff
@@ -110,11 +115,11 @@ func (c *Controller) Reconcile(ctx context.Context, req reconcile.Request) (_ re
 				return
 			}
 
+			log := logf.FromContext(ctx)
```

Comment:
Why are we not using the `logConstructor` for this?

Reply:
I'm already using it one layer above to generate a logger + add the reconcileID afterwards.

If I "re-generate" the logger the reconcileID gets lost